demisto_sdk/commands/common/hook_validations/playbook.py

```diff
@@ -125,6 +126,30 @@ def is_condition_branches_handled(self):  # type: () -> bool
                         task) and is_all_condition_branches_handled
         return is_all_condition_branches_handled
 
+    @error_codes('PB118')
+    def are_all_inputs_in_use(self):  # type: () -> bool
+        """Check whether the playbook inputs are in use in any of the tasks
+
+        Return:
+            bool. if the Playbook inputs are in use.
+        """
+        inputs: Dict = self.current_file.get('inputs', {})
+        inputs_not_in_use = []
+        with open(self.file_path, 'r') as f:
+            playbook_test = f.read()
+        for input in inputs:
+            input_key = input.get('key', '')
```

Comment:
do we have cases without where `key` is not in input?

Reply:
I don't think so, but checking